tldw_Server_API/tests/Chat_NEW/unit/test_provider_model_resolution.py

```
@@ -0,0 +1,45 @@
+import pytest
+
+from tldw_Server_API.app.api.v1.schemas.chat_request_schemas import ChatCompletionRequest
+from tldw_Server_API.app.core.Chat.chat_service import (
+    resolve_provider_and_model,
+    invalidate_model_alias_caches,
+)
+
+
+@pytest.mark.unit
+def test_resolve_provider_and_model_inline_alias(monkeypatch):
```

Ruff (0.14.5)

Unused function argument: `monkeypatch`

(ARG001)